ui/apps/app-center/src/components/app-routes.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+
+import DS from '@akashaproject/design-system';
+// import { useGetLogin, useGetProfile } from '@akashaproject/ui-awf-hooks';
+import { RootComponentProps } from '@akashaproject/ui-awf-typings';
+
+import { useTranslation } from 'react-i18next';
+
+import ExplorePage from './pages/explore-page';
+import InfoPage from './pages/info-page';
+import MyAppsPage from './pages/my-apps-page';
+import AppsPage from './pages/apps-page';
+import WidgetsPage from './pages/widgets-page';
+
+import NavButton from './nav-button';
+
+import routes, { rootRoute, EXPLORE, MY_APPS, APPS, WIDGETS, INFO } from '../routes';
+
+const { Box, Text, BasicCardBox } = DS;
+
+const AppRoutes: React.FC<RootComponentProps> = props => {
+  const { t } = useTranslation();
+
+  return (
+    <Router>
+      <Route path={[routes[EXPLORE], routes[MY_APPS], routes[APPS], routes[WIDGETS]]}>
```

Comment:
I don't understand why the path is an array, what is suppose to do?

Reply:
it matches multiple paths to a component, in this case all these routes share the same card element with tabs

Reply:
I couldn't find any reference for path being an array. You can remove this `<Route>` entirely or use `path="*"` if react-router complains about children not being a `<Route>` element

Reply:
There is an upcoming backwards compatibility layer release for react router v6, so it's likely we won't even need to replace <Switch> with <Routes>, but I don't know the exact details